### ISSUE TITLE
`nettle/tokenizer` is now abandoned

### ISSUE THREAD

**Issue by philiprabbett** [Cloudstek/scim-filter-parser#2]
> Package nette/tokenizer is abandoned, you should avoid using it. No replacement was suggested.

https://github.com/nette/tokenizer

**Comment by mdeboer** [comment]
Good catch! I'll look for a replacement. If you have any suggestions, please let me know!

**Comment by mdeboer** [comment]
@philiprabbett I am closing this issue as I have found a replacement ([Antlr](https://www.antlr.org/)) and currently working on a rewrite.
